packages/opentelemetry-plugin-xml-http-request/README.md

```diff
@@ -0,0 +1,65 @@
+# OpenTelemetry XMLHttpRequest Instrumentation for web
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides auto instrumentation for web using XMLHttpRequest .
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/plugin-xml-http-request
+```
+
+## Usage
+
+```js
+import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { WebTracer } from '@opentelemetry/web';
+import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
+import { ZoneScopeManager } from '@opentelemetry/scope-zone';
+
+const webTracerWithZone = new WebTracer({
+  scopeManager: new ZoneScopeManager(),
+  plugins: [
+    new XMLHttpRequestPlugin({
+      ignoreUrls: ['http://localhost:8090/sockjs-node/info'],
+      propagateTraceHeaderUrls: ['http://localhost:8090']    
```

Comment:
Should this be `propagateTraceHeaderCorsUrls` now that it's renamed? And is this in fact a CORS URL or is this a same-origin one?